### ISSUE TITLE
Add note to content created or edited via third-party OAuth app

### ISSUE THREAD

**Issue by cl8n** [ppy/osu-web#10589]
just got this idea from github:
![](https://github.com/ppy/osu-web/assets/9954349/cc725eb1-a6fd-4bc0-bfef-f21bab338e07)

currently, forum posts and chat messages can be posted by third-party client on behalf of a user, but there is no indication that it came from a third party. I think it'd be good to add a small message or icon that says "posted with \<client name\>" similar to the above image.

**Comment by notbakaneko** [comment]
This is adds a lot of noise imo, especially for chat.

**Comment by cl8n** [comment]
yeah for chat it would need to be just a small/dim icon with tooltip or something. I haven't looked into potential design for this yet.
